src/active_stakers.rs

```diff
@@ -60,10 +59,18 @@ impl ActiveStakers {
         Self::new_with_bounds(bank, DEFAULT_ACTIVE_WINDOW_TICK_LENGTH, bank.tick_height())
     }
 
+    pub fn from_stakes(stakes: Vec<(Pubkey, u64)>) -> Self {
```

Comment:
Motivation for my other comment is to delete this and `stakes()`.  Also, while in the neighborhood, can you rename `ranked_stakes` to `sorted_stakes`?

Reply:
from_stakes() is being used in my test, can I keep it?

Reply:
Burn it. Same with `LeaderSchedule::from_schedule`.  Pass in some bogus stakes instead and `len=2` instead.

Reply:
The stakes object in ActiveStakers is private, so that would force the test to construct a Bank with some dummy votes. That seems more complex than just having a from_stakes() test constructor that can initialize some dummy stakes directly?

Reply:
No, don't do that.  I'm saying don't use ActiveStakers in your LeaderScheduler test.

Reply:
yessir